chrome/android/java/src/org/chromium/chrome/browser/AlwaysOnTopService.java

```diff
@@ -0,0 +1,91 @@
+package org.chromium.chrome.browser;
+
+import android.app.Notification;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.app.Service;
+import android.content.Intent;
+import android.graphics.Color;
+import android.graphics.PixelFormat;
+import android.os.IBinder;
+import android.view.Gravity;
+import android.view.WindowManager;
+import android.widget.ImageView;
+
+import org.chromium.base.Log;
+import org.chromium.chrome.R;
+import org.chromium.chrome.browser.firstrun.FirstRunActivity;
+
+public class AlwaysOnTopService extends Service {
+    private static final String TAG = "AlwaysOnTopService";
+    private static final int NOTIFICATION = R.string.app_name;
+    private static final String APP_NAME = "CASTANETS";
+
+    private ImageView imgView;
+    private NotificationManager mNotificationManager;
+
+    @Override
+    public IBinder onBind(Intent arg0) { return null; }
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        WindowManager mWindowManager = (WindowManager) getSystemService(WINDOW_SERVICE);
```

Comment:
naming style is inconsistent. this variable is local variable.

Reply:
Done. Thanks.